scripts/v6/stored-procs/11-export-bcx-scoa-journal-rollup.sql

```diff
@@ -59,9 +62,9 @@ BEGIN
 			INNER JOIN
 				AssetFinForm aff ON affr.Form_Nr = aff.Form_Nr' end +'
 			LEFT JOIN 
-				SCOAClassification scItem ON sj.BREAKDOWN_SCOA_Item_Debit = scItem.SCOAId
+				SCOAClassification scItem ON sj.BREAKDOWN_SCOA_Item_Debit = scItem.AccountNumber
 			LEFT JOIN 
-				SCOAClassification scProject ON sj.BREAKDOWN_SCOA_Project = scProject.SCOAId
+				SCOAClassification scProject ON sj.BREAKDOWN_SCOA_Project = scProject.AccountNumber
```

Comment:
This is why I didn't necessarily understand the solution of storing the scoa ids the way we do in the Journal to accommodate for the v6 UI. The account number is not unique, so this join can (and very likely will) cause incorrect results. We have just moved the issue from v6 side to v8 side by doing this?

At least we should be adding an additional condition for each of the account number joins which checks the appropriate scoa segment as well, since the account numbers are (until another integration tells us differently) unique **per segment**